control-plane/agents/src/common/errors.rs

```diff
@@ -208,6 +208,11 @@ pub enum SvcError {
         labels: String,
         resource_kind: ResourceKind,
     },
+    #[snafu(display("Invalid {}, labels: {} . Cannot delete `created-by` label as it is used for identifying mayastor pools", resource_kind, labels))]
+    InvalidLabelCreatedBy {
```

Comment:
I don't think we should have an error variant for a specific key. Rather just call it `ForbiddenUnlabelKey` or `RestrictedUnlabelKey` and make it a generic one, for cases we have other labels to restrict. Also replace created-by with {labels} so that it's generic.

Reply:
done